templates/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "<application>"
+version = "0.1.0"
+authors = ["That's you <your@email.com>"]
+
+[dependencies]
+libtock = { git = "https://github.com/tock/libtock-rs" }
+
+[dev-dependencies]
+elf2tab = "0.4.0"
+
+[profile.dev]
+panic = "abort"
+lto = true # Link Time Optimization usually reduces size of binaries and static libraries
```

Comment:
Would the dev profile be used in practice? It may not even be worth configuring.